CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 x.x.x Release notes (yyyy-MM-dd)
 =============================================================
+NOTE: Support for syncing with realm.cloud.io and/or Realm Object Server has been replaced with support for syncing with MongoDB Realm Cloud.
+
+NOTE: This version uses the Realm file format to version 20. It is not possible to downgrade to earlier versions than v10.0.0-beta.13. Non-sync Realms will be upgraded automatically. Synced Realms can only be automatically upgraded if created with Realm JavaScript v10.0.0-beta.1 and above.
+
+### Known issues
```

Comment:
I guess we have other known issues. Perhaps call it "Notable Known Issues" and move it to the end?